pixl_core/src/core/upload.py

```diff
@@ -3,35 +3,47 @@
 import ftplib
 import logging
 import os
+from datetime import datetime
 from ftplib import FTP_TLS
 from pathlib import Path
 
+from core._database import get_project_slug_from_db, update_exported_at_and_save
+
 logger = logging.getLogger(__name__)
 
 # Make a DSHUploader class that takes a project slug and study pseudonymised id?
 
 
 def upload_file(local_file_path: Path) -> str:
+    """Upload local file to hardcoded directory in ftp server."""
+    output_filename = local_file_path.name
+    # Store the file using a binary handler
+    with local_file_path.open("rb") as file_content:
+        upload_content(file_content, output_filename)
+
+
+def upload_content(content: bytes, pseudo_anon_id: str) -> str:
```

Comment:
I think we'd want this to be a generic function which takes in file content and output directory and output filename, rather than doing the database lookup at this point. That way we could reuse it for both writing DICOM files and writing parquet files. Then querying the database for the project happens in a higher level function